data/mapcss-mapping.csv

```diff
@@ -1120,9 +1120,9 @@ wheelchair|yes;1121;
 wheelchair|no;1122;
 wheelchair|limited;1123;
 building|has_parts;1124;
-deprecated|deprecated;1125;x
-deprecated|deprecated;1126;x
-deprecated|deprecated;1127;x
+organic|only;1125;
+organic|yes;1126;
+organic|limited;1127;
```

Comment:
`limited` has **79** (0.37%) uses only and is not mentioned in the wiki even, I don't think we need it.
Maybe merge it into `yes` so that nothing organic is missed

Reply:
Да, можно убрать его

Reply:
"отчасти умеет organic" `organic=yes` покрывает, не?

>organic | yes | The shop or restaurant usually offers some organic products. For  example conventional supermarkets which offer a permanent product line  of organic staple food fall under that category.

а если совсем скудно, то

> organic | no | The shop or restaurant offers no organic products at all or very  little that is basically not worth mentioning (sometimes only used for  marketing purposes for a limited amount of time).

где тут между ними `limited` будет сложно пользователям объяснить
имхо, будут лепить кто как поймёт, кто yes, кто limited...

я бы следовал утверждённому https://wiki.openstreetmap.org/wiki/Proposed_features/organic

Reply:
Ладно с limited. Надо ли специально хранить явное organic=no ?
Мы не храним никакие 'no' сейчас, _кроме_ wheelchair=no

Для его хранения, надо больше коцать код.

Reply:
Так, я напутал. limited можно конвертить в генераторе в yes, он не нужен.
No тоже не нужен.
Yes и only вполне достаточно.

Reply:
done